### PR TITLE
Test_ceph_osd_low_ops_alert backport 4.10

### PR DESCRIPTION
**Pull request by DanielOsypenko**
Cherry pick:
https://github.com/red-hat-storage/ocs-ci/pull/7897

### CODE REVIEW

**Comment by DanielOsypenko** [comment]
> PR validation on existing cluster
> 
> Cluster Name: dosypenk-18-c Cluster Configuration: PR Test Suite: tier3 PR Test Path: tests/manage/monitoring/test_workload_with_distruptions.py::TestCephOSDSlowOps::test_ceph_osd_low_ops_alert Additional Test Params: OCP VERSION: 4.10 OCS VERSION: 4.10 tested against branch: release-4.10
> 
> [Job UNSTABLE](https://url.corp.redhat.com/3e8aef6) (some or all tests failed).

bz: https://bugzilla.redhat.com/show_bug.cgi?id=2228359

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1304,15 +1424,15 @@
 # Users
 NB_SERVICE_ACCOUNT_BASE = "system:serviceaccount:openshift-storage:{}"
 NOOBAA_SERVICE_ACCOUNT_NAME = "noobaa"
-NOOBAA_DB_SERVICE_ACCOUNT_NAME = "noobaa-endpoint"
+NOOBAA_DB_SERVICE_ACCOUNT_NAME = "noobaa-db"
```
Comment (fbalak) [inline ocs_ci/ocs/constants.py:1427]: Why are these changes done as part of backport pr?

**Review by fbalak — CHANGES_REQUESTED** [review]
There seems to be some out of scope changes to constants file

**Comment by DanielOsypenko** [comment]
opened https://github.com/red-hat-storage/ocs-ci/pull/8587 instead

**Pull request closed without merging**